### PR TITLE
add foundation funding

### PR DESCRIPTION
**Pull request by jharenza**
This PR adds CBTN foundation funding.

### CODE REVIEW

File: content/05.administrative.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ## Acknowledgments
 
-We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible.
+We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network (CBTN) and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible.
+
+Philanthropic support has ensured the CBTN's ability to collect, store, manage, and distribute specimen and data since it was founded in 2013. 
+In addition to the support from the CBTN Executive Council members and Brain Tumor Board of Visitors, the following donors have provided leadership level support for CBTN: Children's Brain Tumor Foundation, Easie Family Foundation, Kortney Rose Foundation, Lilabean Foundation, Minnick Family Charitable Fund, Perricelli Family, Pslam 103 Foundation, and Swifty Foundation.
```
Comment (jaclyn-taroni) [inline content/05.administrative.md:6]:
```suggestion
In addition to the support from the CBTN Executive Council members and Brain Tumor Board of Visitors, the following donors have provided leadership level support for CBTN: Children's Brain Tumor Foundation, Easie Family Foundation, Kortney Rose Foundation, Lilabean Foundation, Minnick Family Charitable Fund, Perricelli Family, Psalm 103 Foundation, and Swifty Foundation.
```

File: build/assets/custom-dictionary.txt [hunk 2]
```diff
@@ -586,6 +592,7 @@ Priya
 Prolo
 proteogenomic
 pseudoalignment
+Pslam
```
Comment (jaclyn-taroni) [inline build/assets/custom-dictionary.txt:595]: You might not need this anymore. I would not be surprised if `psalm` was in the dictionary.

```suggestion
```

**Review by jaclyn-taroni — APPROVED** [review]
One spelling error that needs to be corrected in the admin section and the custom dictionary.

File: content/05.administrative.md (hunk 1 above)
Comment (jharenza) [reply content/05.administrative.md:6]: oh yes, looks like that is it - just copied and pasted this from someone.

**Comment by jharenza** [comment]
@jaclyn-taroni just wanted to let you know I found one more N 1043-->1074 which I updated in the results section here

**Comment by jharenza** [comment]
@sjspielman or @jashapiro can one of you review the non-foundation changes in this PR  since @jaclyn-taroni is out (kept all in this last PR so ready to submit tonight once this goes in)

File: content/04.discussion.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ Using medulloblastoma subtyping and immune deconvolution analyses performed here
...
 Moreover, OpenPBTA has enabled a framework to support real-time integration of clinical trial subjects as each was enrolled on the PNOC008 high-grade glioma clinical trial [@clinicaltrials:NCT03739372], allowing researchers and clinicians to link tumor biology to translational impact through clinical decision support during tumor board discussions.
-Finally, as part of the the NCI's Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort (https://github.com/PediatricOpenTargets/OpenPedCan-analysis) to build the Molecular Targets Platform (https://moleculartargets.ccdi.cancer.gov/) in support of the RACE Act.
+Finally, as part of the the NCI's Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort [https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis) to build the Molecular Targets Platform ([https://moleculartargets.ccdi.cancer.gov/](https://moleculartargets.ccdi.cancer.gov/)) in support of the RACE Act.
```
Comment (sjspielman) [inline content/04.discussion.md:39]: I might still put the link in parentheses - 

```suggestion
Finally, as part of the the NCI's Childhood Cancer Data Initiative (CCDI), the OpenPBTA project was recently expanded into a pan-pediatric cancer effort ([https://github.com/PediatricOpenTargets/OpenPedCan-analysis](https://github.com/PediatricOpenTargets/OpenPedCan-analysis)) to build the Molecular Targets Platform ([https://moleculartargets.ccdi.cancer.gov/](https://moleculartargets.ccdi.cancer.gov/)) in support of the RACE Act.
```

File: content/05.administrative.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ## Acknowledgments
 
-We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible.
+We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network (CBTN) and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible.
```
Comment (sjspielman) [inline content/05.administrative.md:3]:
```suggestion
We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network (CBTN) and/or the Pacific Pediatric Neuro-oncology Consortium, without which this research would not be possible.
```

File: content/03.results.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/ne
...
 Since the CBTN opened its collection protocol in 2011, before molecular data were integrated into classifications, the majority of the samples within the OpenPBTA lacked molecular subtype annotations at the time of tissue collection.
 Moreover, the OpenPBTA data does not yet feature methylation arrays which are increasingly used to inform molecular subtyping.
-Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to generate research-grade integrated diagnoses for 59% (630/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
+Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to generate research-grade integrated diagnoses for 60% (641/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
```
Comment (sjspielman) [inline content/03.results.md:41]: Can you quickly point me to where this update from 630 --> 641 came from?

Comment (jharenza) [reply content/03.results.md:41]: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/sample-distribution-analysis/03-tumor-descriptor-and-assay-count.nb.html

**Review by sjspielman — APPROVED** [review]
don't need to see again!

File: content/03.results.md (hunk 5 above)
Comment (jashapiro) [reply content/03.results.md:41]: Ther 59%->60% change should also be made in the discussion, line 13.

Comment (jashapiro) [inline content/03.results.md:41]:
```suggestion
Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review, to generate research-grade integrated diagnoses for 60% (641/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
```

**Review by jashapiro — APPROVED** [review]
LGTM, with the caveat of a number update needed in the discussion that I can't directly comment on. (line 13), and a comma added below.

**Comment by jharenza** [comment]
Thanks, both! will update this afternoon.

File: content/03.results.md (hunk 5 above)
Comment (jharenza) [reply content/03.results.md]: done! thanks for catching that!

**Pull request merged**